docs/skills/odqa.rst

```diff
@@ -80,14 +80,17 @@ Comparison
 
 Scores for **ODQA** skill:
 
-
-+-----------------------+-----------------------------+----------------+-----------------------+--------+------+
-| Model                                               | Dataset        |  Wiki dump            |  F1    |  EM  |
-+-----------------------------------------------------+----------------+-----------------------+--------+------+
-|:config:`DeepPavlov <odqa/en_odqa_infer_wiki.json>`  | SQuAD (dev)    |   enwiki (2018-02-11) |  28.0  | 22.2 |
-+-----------------------------------------------------+----------------+-----------------------+--------+------+
-|`DrQA`_                                              | SQuAD (dev)    |   enwiki (2016-12-21) |   \-   | 27.1 |
-+-----------------------------------------------------+----------------+-----------------------+--------+------+
++-----------------------------------------------------+----------------+---------------------+---------------------+
+|                                                     |                | enwiki (2018-02-11) | enwiki (2016-12-21) |
+|                                                     |                +----------+----------+-----------+---------+
+| Model                                               | Dataset        |  F1      |   EM     |   F1      |   EM    |
```

Comment:
If `Dataset` is the same in all rows, maybe it would be better to move it to the table description?

Reply:
This table will be extended soon with other datasets (TriviaQA and WikiMovies).